package.json

```diff
@@ -9,7 +9,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint --fix",
-    "validate-md": "node scripts/validateMarkdown.js"
+    "validate-md": "node scripts/validateMarkdown.js",
+    "test": "npx percy exec -- cypress run"
```

Comment:
Just one question/comment: Do we need to use `npx percy` since we installed `@percy/cli` below?

Reply:
Fixed to `percy exec -- cypress run` 👍

Reply:
👌